control-plane-operator/controllers/hostedcontrolplane/configoperator/reconcile.go

```diff
@@ -314,6 +314,9 @@ func buildHCCContainerMain(image, hcpName, openShiftVersion, kubeVersion string,
 			fmt.Sprintf("--oauth-address=%s", oauthAddress),
 			fmt.Sprintf("--oauth-port=%d", oauthPort),
 		}
+		if platformType == hyperv1.IBMCloudPlatform {
+			c.Command = append(c.Command, "--controllers=controller-manager-ca,resources,inplaceupgrader,drainer,hcpstatus")
```

Comment:
If we are setting this: I don't believe we need inplaceupgrader and drainer do we?